src/components/BreadcrumbMenu/BreadcrumbMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink, useParams, matchPath } from 'react-router-dom';
+import { HiMenuAlt3 } from 'react-icons/hi';
+import useBreadcrumbs from 'use-react-router-breadcrumbs';
+
+const DynamicBreadcrumb = () => {
+  const { id } = useParams();
+  return <span>{id}</span>;
+};
+
+const routes = [
+  { path: '/', breadcrumb: 'Home' },
+  { path: '/admin', breadcrumb: null },
+  { path: '/admin/cohorts/:id', breadcrumb: DynamicBreadcrumb },
+];
+
+const excludePaths = ['/', '/admin/cohorts/:id'];
+
+const isRouteExcluded = (pathname: string) => {
+  return excludePaths.find((path) => matchPath({ path }, pathname));
+};
```

Comment:
maybe we should move all these to a separate file in the folder, called `utils.tsx`

Reply:
@jpatrici I did make an effort to do this but it was severely breaking stuff and I couldn't figure out what was wrong. I left it as-is for now until someone can take a look!

Reply:
@RDiMag I've moved them to the utils file :)